dbus/src/nonblock/generated_org_freedesktop_dbus.rs

```diff
@@ -0,0 +1,209 @@
+// This code was autogenerated with `dbus-codegen-rust -d org.freedesktop.DBus -p /org/freedesktop/DBus -m none -c nonblock -g -i org.freedesktop. --dbuscrate crate --filter DBus -o dbus/src/nonblock/generated_org_freedesktop_dbus.rs`, see https://github.com/diwic/dbus-rs
+#![allow(missing_docs)]
```

Comment:
This is currently not automatically added. It fails for me for files that are `include!(..)`d, like in the dbus-codegen tests. But it is generally useful for those auto generated files.

Should this be a generator option? (defaulting to add `#![allow(missing_docs)]`)

Reply:
Another option could be to actually insert some docs, like "This is the `Foo` signal on the `bar.baz` interface".

Reply:
Those would be very generic. A dbus introspection response may actually have annotations for this purpose, you probably know that already. But no service is really using them. The question is, if it is worth parsing those annotations and what to do with all the non-annotated methods and interfaces.